build/includes/sdk.mk

```diff
@@ -139,7 +139,7 @@ run-sdk-conformance-no-build: DELAY ?= $(shell bash -c "echo $$[ ($(RANDOM) % 5
 run-sdk-conformance-no-build: TESTS ?= $(DEFAULT_CONFORMANCE_TESTS)
 run-sdk-conformance-no-build: GRPC_PORT ?= 9357
 run-sdk-conformance-no-build: HTTP_PORT ?= 9358
-run-sdk-conformance-no-build: FEATURE_GATES ?=
+run-sdk-conformance-no-build: FEATURE_GATES := ""
```

Comment:
Curious - why this change? Don't we want it to be overwritable?

Reply:
I consider it as the easiest solution at the moment. Probably it is worth adding a comment as we do in _terraform.mk_ - "All Feature gates are disabled". What do you think?

Reply:
But we need to be able to pass feature flags to conformance tests. We can't remove that ability. In fact, this will turn off tests for the Go conformance tests, so we can't do this.

We have to use `?=`, so that it can be overwritten.

Reply:
https://www.gnu.org/software/make/manual/html_node/Setting.html 

If you want to understand the different assignments in Make.